tests/test_modeling_common.py

```diff
@@ -3846,6 +3855,37 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
                                     if "attention_mask" in inspect.signature(model_eager.forward).parameters:
                                         processed_inputs["attention_mask"] = dummy_attention_mask
 
+                                if (
+                                    "bool_masked_pos" in inspect.signature(model_eager.forward).parameters
+                                ) and not deactivate_mask:
```

Comment:
nit - let's check deactivate_mask first, it'll be quicker 

```suggestion
                                if not deactivate_mask and (
                                    "bool_masked_pos" in inspect.signature(model_eager.forward).parameters
                                ):
```